examples/clustermanager/simple/UnixDomainCM.jl

```diff
@@ -9,7 +9,7 @@ function launch(manager::UnixDomainCM, params::Dict, launched::Array, c::Conditi
     for i in 1:manager.np
         sockname = tempname()
         try
-            cmd = `$(params[:exename]) --worker=custom $(@__FILE__) worker $sockname`
```

Comment:
Did the filename have a "-" character in it?  Do you have any examples of what the arguments were that caused this to happen?